src/blocks/content-helper/content-helper.tsx

```diff
@@ -9,8 +9,8 @@ import { registerPlugin } from '@wordpress/plugins';
 /**
  * Internal dependencies
  */
-import CurrentPostDetails from './current-post-details/component';
-import RelatedTopPostList from './components/related-top-post-list';
+import PerformanceDetails from './performance-details/component';
+import RelatedTopPostList from './related-top-performing-posts/component-list';
```

Comment:
Here `related-top-posts` seems better than `related-top-performing-posts`.

Reply:
I've renamed `related-top-performing-posts` to `related-top-posts` as suggested. I did not rename with Angular naming conventions but we can do so in the future if needed.

I'll soon mark this PR as ready for review.